packages/perseus-editor/src/components/scrollless-number-text-field.tsx

```diff
@@ -39,7 +49,33 @@ const ScrolllessNumberTextField = (props: PropsFor<typeof TextField>) => {
         };
     }, [inputRef]);
 
-    return <TextField type="number" {...props} ref={inputRef} />;
+    return (
+        <TextField
+            {...restOfProps}
+            type="number"
+            value={focused ? wipValue : value}
+            onChange={(newValue) => {
+                setWipValue(newValue);
+                onChange(newValue);
+            }}
+            onFocus={(e) => {
+                setWipValue(value);
+                setFocused(true);
+
+                if (props.onFocus) {
+                    props.onFocus(e);
+                }
```

Comment:
Could this be simplified to

```suggestion
                props.onFocus?.(e);
```